test/unitTests/OmnisharpPackageCreator.test.ts

```diff
@@ -31,66 +31,66 @@ suite("GetOmnisharpPackage : Output package depends on the input package and oth
 
         test(`Architectures, binaries and platforms do not change useFramework: ${useFramework}`, () => {
             let testPackage = inputPackages.find(element => (element.platformId && element.platformId == "os-architecture"));
-            let resultPackage = SetBinaryAndGetPackage(testPackage, useFramework, serverUrl, version, installPath);
+            let resultPackage = SetBinaryAndGetPackage(testPackage!, useFramework, serverUrl, version, installPath);
```

Comment:
Could add a helper such as the one in this [TS playground](https://www.typescriptlang.org/play?#code/DYUwLgBAbghsCuIIC4IDt4FsBGIBOEAPuvMMERPGgCYgBmAlmiNRALwQDMA3AFAMBnAHIB7MENLAAFLAQgAlH14BjEWgEjQAOmAiA5lIBEsxBEEQB4AISHFvXiaQcqtRs2pLBo8ZJlxEdrx0VMpgDGpmwmISZAA8ACoAfH5yqPHyKBAwApZ4YALQ-kjmomgxwDDYoAmJEADevBBNZnQQKaZsnSRkRMSO7F0u9EwsGQ3NExBgABZ4IgDu6CCLAKJ4c3hG-eYYPSIEQ24sWrZ8EwC+vOdAA), then the compiler would be happy.

```typescript
function isNotNull<T>(value: T) : asserts value is NonNullable<T> {
    if (value === null || value === undefined) {
        throw new Error("value is null or undefined.");
    }
}
```

Reply:
I added this, and updated all the places that I thought made sense to use it
1.  Places that were asserting not null using the non-annotated APIs were replaced with this function
2.  Places where the nullable item was used multiple times (and would have required multiple `!`).

Otherwise, I left the `!` usage when the nullable item was used only once